zfs/pool.go

```diff
@@ -68,17 +71,23 @@ func poolNames() ([]string, error) {
 	if err != nil {
 		return nil, err
 	}
+	stderr, err := cmd.StderrPipe()
+	if err != nil {
+		return nil, err
+	}
 	scanner := bufio.NewScanner(out)
 
 	if err = cmd.Start(); err != nil {
-		return nil, err
+		return nil, errors.Wrapf(err, "Failed to start command %s", cmd.String())
```

Comment:
Replace `errors.Wrapf` with `fmt.Errorf("Failed to start command '%s': %w", cmd.String(), err)` to avoid additional dependency - error wrapping is in the stdlib.

Reply:
Done